cores/esp8266/time.cpp

```diff
@@ -198,15 +198,43 @@ void setTZ(const char* tz){
 void configTime(const char* tz, const char* server1, const char* server2, const char* server3)
 {
     sntp_stop();
-
     setServer(0, server1);
     setServer(1, server2);
     setServer(2, server3);
-	setTZ(tz);
-	
+    setTZ(tz);
     sntp_init();
 }
 
+/*
+ * Backported from ESP32 Arduino
+ */
+
+bool getLocalTime(struct tm * info, uint32_t ms) 
+{
+  uint32_t count = ms ;
+  time_t now;
+ 
+  time(&now);
+  localtime_r(&now, info);
+ 
+  if (info->tm_year > (2016 - 1900)) 
+  {
+    return true;
+  }
+ 
+  while (count--)
```

Comment:
With this loop, maximum timeout is 10 times the value of `ms`, which is why Earle divided it by 10.
Original and probably later updated / fixed esp32 code [is there](https://github.com/espressif/arduino-esp32/blob/34125cee1d1c6a81cd1da07de27ce69d851f9389/cores/esp32/esp32-hal-time.c#L91) with a more accurate loop.
Do you think you can improve your proposal ?

Reply:
Okay then, do you have an idea how LittleFS_Timestamp.ino sketch should be modified?

Reply:
I think it is OK to use the esp32 implementation globally and remove littlefs test's one.